skfp/fingerprints/e3fp_fp.py

```diff
@@ -91,15 +93,23 @@ def _calculate_single_mol_fingerprint(
         # Generating conformers
         # TODO: for some molecules conformers are not properly generated - returns an empty list and throws RuntimeError
         try:
-            mol, values = conf_gen.generate_conformers(mol)
-            fps = fprints_from_mol(
-                mol,
-                fprint_params={
-                    "bits": self.n_bits_before_hash,
-                    "radius_multiplier": self.radius_multiplier,
-                    "rdkit_invariants": self.rdkit_invariants,
-                },
-            )
+            try:
```

Comment:
Please add a switch instead of disabling it at all

Reply:
+1